cpu/atmega_common/periph/gpio.c

```diff
@@ -174,11 +174,21 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
 
     /* apply flank to interrupt number int_num */
     if (int_num < 4) {
-        EICRA |= (flank << (int_num * 2));
+        if (flank == GPIO_LOW) {
```

Comment:
This issue also exists with all flanks that contain zeros (i.e. anything but GPIO_RISING).

I would suggest removing the check and instead always zero the relevant bits, followed by writing the flank onto them.